### PR TITLE
Swap meta fix

### PR DESCRIPTION
**Pull request by laics1984**
- Fixes on token balance
- metamask account changes behavior

### CODE REVIEW

File: src/modules/services/submodule/mainnetSwap/submodule2/bsc/views/ViewServicesMainnetSwapBscOptions.vue [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="flex justify-between text-sm">
     <div><span class="text-gray-400">Swap > BSC > </span> <span class="text-blue-primary font-bold">Select option</span></div>
     <div>
-      <router-link :to="{ name: 'ViewServices'}" class="font-bold" active-class="accounts">All Services</router-link>
+      <router-link :to="{ name: 'ViewServices'}" class="font-bold" active-class="accounts">Home</router-link>
```
Comment (sleepyOwl14) [inline src/modules/services/submodule/mainnetSwap/submodule2/bsc/views/ViewServicesMainnetSwapBscOptions.vue:5]: this Home can be apply to the ts-base ?

File: src/modules/services/submodule/mainnetSwap/submodule2/eth/views/ViewServicesMainnetSwapEthOptions.vue [hunk 2]
```diff
@@ -2,7 +2,7 @@
   <div class="flex justify-between text-sm">
     <div><span class="text-gray-400">Swap > ETH > </span> <span class="text-blue-primary font-bold">Select option</span></div>
     <div>
-      <router-link :to="{ name: 'ViewServices'}" class="font-bold" active-class="accounts">All Services</router-link>
+      <router-link :to="{ name: 'ViewServices'}" class="font-bold" active-class="accounts">Home</router-link>
```
Comment (sleepyOwl14) [inline src/modules/services/submodule/mainnetSwap/submodule2/eth/views/ViewServicesMainnetSwapEthOptions.vue:5]: this also, same question as above

File: src/modules/services/submodule/mainnetSwap/submodule2/bsc/views/ViewServicesMainnetSwapBscOptions.vue (hunk 1 above)
Comment (sleepyOwl14) [reply src/modules/services/submodule/mainnetSwap/submodule2/bsc/views/ViewServicesMainnetSwapBscOptions.vue:5]: @laics1984 please confirm, thanks

Comment (laics1984) [reply src/modules/services/submodule/mainnetSwap/submodule2/bsc/views/ViewServicesMainnetSwapBscOptions.vue]: @sleepyOwl14 Title has been updated. Thanks

File: src/modules/services/submodule/mainnetSwap/submodule2/eth/views/ViewServicesMainnetSwapEthOptions.vue (hunk 2 above)
Comment (laics1984) [reply src/modules/services/submodule/mainnetSwap/submodule2/eth/views/ViewServicesMainnetSwapEthOptions.vue]: @sleepyOwl14 Title has been updated. Thanks

**Review by sleepyOwl14 — APPROVED** [review]
all good

**Comment by laics1984** [comment]
> all good

Thanks @sleepyOwl14 , will squash and merge

**Pull request merged**